### PR TITLE
Zold payments

### PR DESCRIPTION
**Pull request by g4s8**
#1074 - allow to setup zold-wallets for users, add `ZoldBank` with `@todo` to implement.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1075](https://github.com/zerocracy/farm/issues/1075) is now in scope, role is `REV`

File: src/main/java/com/zerocracy/pmo/banks/Zold.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+public final class Zold implements Bank {
+    @Override
+    public Cash fee(final Cash amount) throws IOException {
+        return Cash.ZERO;
+    }
+
+    @Override
+    public String pay(final String target, final Cash amount,
+        final String details) throws IOException {
+        throw new IOException("Zold payments are not implemented yet");
```
Comment (carlosmiranda) [inline src/main/java/com/zerocracy/pmo/banks/Zold.java]: @g4s8 Let's throw `UnsupportedOperationException` instead, to easily distinguish whether we're calling this accidentally when we shouldn't yet.

**Comment by g4s8** [comment]
@carlosmiranda fixed, please take a look: c5a92f7e70daf4cf34b599dd8ca3ebd10bc0b4d8

**Comment by yegor256** [comment]
@g4s8 let's fix your TODO and point it to this repo: git@github.com:zold-io/java-api.git We will implement the API and then will use it in the farm.

**Comment by g4s8** [comment]
@yegor256 fixed todo, please take a look https://github.com/zerocracy/farm/pull/1075/commits/fb9e304e50e5d1da5f29b300460e3e6b5e146236

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1075](https://github.com/zerocracy/farm/issues/1075) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by ypshenychka** [comment]
@g4s8 According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>All commit messages have links to issues.

Commit with review fix contains link to incorrect issue. Please confirm that you'll try to avoid this mistake next time.

**Comment by g4s8** [comment]
@ypshenychka I confirm

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@g4s8 @carlosmiranda Thank you both!

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**